tpm2/structures.go

```diff
@@ -688,7 +688,12 @@ func decodeQuoteInfo(in *bytes.Buffer) (*QuoteInfo, error) {
 	if err != nil {
 		return nil, fmt.Errorf("decoding PCRSelection: %v", err)
 	}
-	out.PCRSelection = sel
+	if len(sel) == 1 {
+		out.PCRSelection = sel[0]
+	} else {
+		return nil, fmt.Errorf("PCRSelection is empty")
```

Comment:
It could also have >1 PCR selection. How about something like:
```
if len(sel) != 1 {
  return nil, fmt.Errorf("got %d PCRSelections, expected 1", len(sel))
}
out.PCRSelection = sel[0]
```